go/cli/mcap/cmd/doctor.go

```diff
@@ -261,8 +261,12 @@ func (doctor *mcapDoctor) Examine() error {
 				doctor.error("Failed to parse schema:", err)
 			}
 
-			if schema.Encoding == "" && len(schema.Data) > 0 {
-				doctor.error("Schema.data field should not be set when Schema.encoding is empty")
+			if schema.Encoding == "" {
+				if len(schema.Data) == 0 {
+					doctor.warn("Schema record with ID: %d name: %d is empty", schema.ID, schema.Name)
```

Comment:
"is empty" might be ambiguous. Probably better to be explicit that the `data field is empty`